fluent-react/examples/html-markup/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Localized, withLocalization } from 'fluent-react';
+
+function App(props) {
+  function showAlert(id) {
+    const { getString } = props;
+    alert(getString(id));
+  }
+
+  return (
+    <div>
+      <Localized id="sign-in-or-cancel"
+        button={<button onClick={() => showAlert('clicked-sign-in')}></button>}
+        a={<button className="text" onClick={() => showAlert('clicked-cancel')}></button>}
```

Comment:
Yes, the result will be two buttons.

I went for names that are familiar if you know HTML. Otherwise we'd need something like `<button1>` and `<button2>`, or `<signin>` and `<cancel>`. I'm slightly worried about using non-standard HTML names in the translations because the are more likely to be translated by the localizers.

Let's also keep in mind that it's rare to have two elements of the same type.